dump/dump_konnect_test.go

```diff
@@ -280,3 +280,63 @@ func Test_filterNonKongPackages(t *testing.T) {
 		})
 	}
 }
+
+func Test_excludeKonnectManagedPlugins(t *testing.T) {
+	tests := []struct {
+		name    string
+		plugins []*kong.Plugin
+		want    []*kong.Plugin
+	}{
+		{
+			name: "eclude konnect tags",
```

Comment:
```suggestion
			name: "exclude konnect tags",
```

We should enable a spell check linter to automate this.